internal/lib/lib.go

```diff
@@ -1127,17 +1127,15 @@ func GetClusterID() string {
 
 func GetClusterIDSplit() string {
 	clusterID := GetClusterID()
-	if clusterID != "" {
-		clusterName := strings.Split(clusterID, ":")
-		if len(clusterName) > 1 {
-			if GetVPCMode() {
-				// Include first 5 characters to add more uniqueness to cluster name
-				return clusterName[0] + "-" + clusterName[1][:5]
-			}
-			return clusterName[0]
+	clusterName := strings.Split(clusterID, ":")
+	if len(clusterName) > 1 {
+		if GetVPCMode() {
+			// Include first 5 characters to add more uniqueness to cluster name
+			return clusterName[0] + "-" + clusterName[1][:5]
 		}
+		return clusterName[0]
 	}
-	return ""
+	return clusterID[:12]
```

Comment:
What is max length of clusterId? Can it go out of bound?

Reply:
Done.